### PR TITLE
Updated Department Code List

### PR DESCRIPTION
**Pull request by abhinavjain241**
@icyflame : Please check.

### CODE REVIEW

**Comment by icyflame** [comment]
@abhinavjain241 thanks a lot!

Quick question: is the code for CORAL N2? Meaning it has a digit? If this is the case, then I have to change the regex that I am using in the script as well.

File: upload-and-update/departments.json [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+   "IM": "Industrial & Systems Engineering",
+   "IT": "Information Technology",
+   "MS": "Materials Science",
+   "MA": "Mathematics",
+   "ME": "Mechanical Engineering",
+   "MD": "Medical Science & Technology",
+   "MT": "Metallurgical & Materials Engineering",
+   "MI": "Mining Engineering",
+   "NA": "Ocean Engineering & Naval Architecture",
+   "N2": "Oceans, Rivers, Atmosphere and Land Sciences",
```
Comment (icyflame) [inline upload-and-update/departments.json:26]: For coral, I found [this](http://www.iitkgp.ac.in/downloads/coral.pdf)., and it seems subject names start with `CL`.

Comment (abhinavjain241) [inline upload-and-update/departments.json:26]: @icyflame : Yeah, I guess you can change that to `CL`, there were other departments also, which had different ID's in the hyperlinks, which I changed manually, I didn't know about this, but this seems right.

Comment (icyflame) [inline upload-and-update/departments.json:26]: @abhinavjain241 Okay, I will change that merge.

**Comment by icyflame** [comment]
Merged in 4165187

**Pull request closed without merging**